mods/cnc/maps/funpark01/map.yaml

```diff
@@ -0,0 +1,544 @@
+Selectable: False
+
+MapFormat: 6
```

Comment:
The map format should be updated.
But I think this happens automatically when starting the game.